paymentsheet/src/test/java/com/stripe/android/customersheet/CustomerSheetTest.kt

```diff
@@ -142,6 +143,37 @@ class CustomerSheetTest {
             assertThat(selectedResult.selection).isNull()
         }
 
+    @Test
+    fun `On configure, should persist on config changes & process death`() = runTestActivityTest {
```

Comment:
```suggestion
    fun `On configure, should persist on config changes`() = runTestActivityTest {
```

We're not testing process death here.